build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
+"""
```

Comment:
I'll defer to Poldrackgroup on this. Personally I'm trying to keep anything new compatible with python3. Sorry to hear traits isn't playing well with your python3 environment Vanessa.